src/js/background.js

```diff
@@ -135,8 +135,13 @@ if (!config.toggle) {
 const blacklist = 'https://raw.githubusercontent.com/keraf/NoCoin/master/src/blacklist.txt';
 fetch(blacklist)
     .then(resp => {
-        if (resp.status === 200) {
-            resp.text().then(text => runBlocker(text));
+        if (resp.status === 200 ) {
+            resp.text().then((text) => {
+				if(text === "" || text == null)
```

Comment:
Just a few notes regarding the code:
- The indentation looks incorrect. Make sure you are using 4 spaces.
- Text should always be a string and therefore null checking is not necessary.
- For consistency, use single quotes across the project.
- For consistency again, use curly brackets for all conditions.

I will add linting rules in the future.